rqt_controller_manager/src/rqt_controller_manager/controller_manager.py

```diff
@@ -383,7 +383,11 @@ def data(self, index, role):
 
         if role == Qt.DecorationRole:
             if index.column() == 0:
-                return self._icons[ctrl.state]
+                
+                try:                                
+                    return self._icons[ctrl.state]  #KeyError: 'initialized'
+                except:
```

Comment:
shouldn't we specifically catch the `KeyError` you put in the comment?

Reply:
Actually, there is a missing `ctrl.state`: `initialised`